.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,55 @@
+name: pre-commit
```

Comment:
This file can be removed as the pre-commit workflow is disabled/should be completly removed. Sorry for the inconvenience.